js/graphql.md

```diff
@@ -0,0 +1,1346 @@
+---
+---
+# The GraphQL Transform
+
+The GraphQL Transform is a library that simplifies the process of developing, deploying, and maintaining GraphQL APIs. With it, you define your API using the GraphQL Schema Definition Language (SDL) and can then use this library where to transform it into a fully descriptive cloudformation template that implements the API's data model.
+
+The Transform can be run as an independent library, however it is integrated into the Amplify CLI via the `API` category for you to use immediately.
+
+## Quick Start
+
+Navigate into the root of a JavaScript, iOS, or Android project and run:
+
+```bash
+amplify init
+```
+
+Follow the wizard to create a new app. After finishing the wizard run:
+
+```bash
+amplify add api
+
+# Select the graphql option and when asked if you 
+# have a schema, say No.
+# Select one of the default samples. You can change it later.
+# Choose to edit the schema and it will open your schema.graphql in your editor.
+```
+
+You can leave the sample as is or try this schema.
+
+```
+type Blog @model {
+  id: ID!
+  name: String!
+  posts: [Post] @connection(name: "BlogPosts")
+}
+type Post @model {
+  id: ID!
+  title: String!
+  blog: Blog @connection(name: "BlogPosts")
+  comments: [Comment] @connection(name: "PostComments")
+}
+type Comment @model {
+  id: ID!
+  content: String
+  post: Post @connection(name: "PostComments")
+}
+```
+
+Once you are happy with your schema, save the file and click enter in your
+terminal window. If now error messages are thrown then you are good to go
```

Comment:
"if **no** error messages are thrown"

Also maybe change this to, "if no error messages are thrown this means the transformation was successful and you can deploy your new API" just to make it very clear what's happened? And slightly less informal.

Reply:
+1